client/src/components/AuditAdmin/Progress/JurisdictionDetail.tsx

```diff
@@ -274,34 +319,49 @@ const RoundStatusSection = ({
     }
 
     if (sampleCount.ballots === 0) return <p>No ballots sampled</p>
-    if (jurisdictionStatus === JurisdictionRoundStatus.COMPLETE)
-      if (auditSettings.auditType === 'BATCH_COMPARISON')
+    if (jurisdictionStatus === JurisdictionRoundStatus.COMPLETE) {
+      if (auditSettings.auditType === 'BATCH_COMPARISON') {
         return (
           <div>
             <p>Results finalized</p>
             <AsyncButton
-              onClick={async () => {
-                if (
-                  await unfinalizeBatchResults(
-                    electionId,
-                    jurisdiction.id,
-                    round.id
-                  )
-                )
-                  // Hack: for now just reload the whole page here instead of
-                  // properly refreshing the state from the server, since the
-                  // state is way high up in the component hierarchy
-                  window.location.reload()
-              }}
+              onClick={() =>
+                unfinalizeBatchResults({
+                  electionId,
+                  jurisdictionId: jurisdiction.id,
+                  roundId: round.id,
+                })
+              }
```

Comment:
I noticed that we don't disable the existing "Unfinalize Results" button when the current round is completed. Clicking the button surfaces an error that provides a brief explanation. I followed that same pattern for the new "Reopen" buttons:

![error](https://user-images.githubusercontent.com/12616928/186724053-dbf2c9e1-6603-416c-aa9e-6d9d186a496f.png)

Open to changing this behavior, of course! We could add a suggestion to the error message to reach out to a support user to reopen the round. Or we could disable the button and add an explanatory tooltip

Reply:
This pattern seems good enough for now.

I think adding instructions about contacting support is a bit tricky in Arlo because VotingWorks isn't the only host for Arlo - some states host it themselves. I've never quite known what to do so I have avoided putting specific instructions in the app.l

Reply:
Ah yes good point about some states hosting Arlo themselves - will leave as is